scripts/prepare_sector_network.py

```diff
@@ -3603,7 +3737,7 @@ def add_industry(n, costs):
     n.add(
         "Link",
         nodes + " Fischer-Tropsch",
-        bus0=nodes + " H2",
+        bus0=spatial.h2.nodes, #nodes + " H2",
```

Comment:
I think this is still an artefact from when we thought about not resolving the hydrogen network
Maybe switch back to `nodes + " H2"`